### PR TITLE
Fix for #Pydev 993: isort is not executed from the project folder

### PR DESCRIPTION
**Pull request by oluiscabral**

### CODE REVIEW

File: plugins/org.python.pydev.jython/src/org/python/pydev/jython/JythonModules.java [hunk 1]
```diff
@@ -52,13 +52,16 @@ private static PyObject getPepJythonModule(IPythonInterpreter interpreter, Strin
...
+        String outputLine = "output = getattr(isort.SortImports(file_contents=fileContents, settings_path=settingsPath, known_third_party=knownThirdParty), 'output', None)\n";
         if (iPythonInterpreter == null) {
             // The first call may be slow because doing the imports is slow, but subsequent calls should be
             // fast as we'll be reusing the same interpreter.
             String s = ""
                     + "import sys\n"
+                    + "import os\n"
                     + "add_to_pythonpath = '%s'\n"
+                    + "os.chdir(add_to_pythonpath)\n"
+                    + "add_to_pythonpath = os.getcwd()\n"
```
Comment (fabioz) [inline plugins/org.python.pydev.jython/src/org/python/pydev/jython/JythonModules.java:64]: I don't think this line is needed (i.e.: `add_to_pythonpath = os.getcwd()`).

If it's needed, can you elaborate a bit on it?

Comment (oluiscabral) [reply plugins/org.python.pydev.jython/src/org/python/pydev/jython/JythonModules.java:64]: Following https://www.tutorialspoint.com/python/os_chdir.htm use example, os.chdir(path) goes to the respectively path directory and os.getcwd() return it's path

i.e.: os.chdir(add_to_pythonpath) by itself does not change add_to_pythonpath

Comment (oluiscabral) [reply plugins/org.python.pydev.jython/src/org/python/pydev/jython/JythonModules.java]: Correction:
I did some tests and perhaps os.getcwd() is kinda redundant, so I remove it from the fix

**Comment by fabioz** [comment]
Just merged. Thanks for the fix ;)

**Pull request merged**